pymongo_auth_aws/auth.py

```diff
@@ -45,6 +47,26 @@ def _aws_temp_credentials():
     if access_key and secret_key:
         return AwsCredential(
             access_key, secret_key, os.environ.get('AWS_SESSION_TOKEN'))
+    # Check if environment variables exposed by IRSA are present
```

Comment:
Can you please spell out the acronym here?

Reply:
Nit: please end comments with a period to match existing style.

Reply:
Could you also add a comment with a link to the Amazon docs where the IRSA auth process is defined?